config/initializers/5_telemetry.rb

```diff
@@ -1,3 +1,11 @@
+# Set multiprocess-friendly data store
+require 'prometheus/client'
+require 'prometheus/client/data_stores/direct_file_store'
+PROMETHEUS_STORE_DIR = File.join(Rails.root, 'tmp', 'prometheus')
+Dir.mkdir(PROMETHEUS_STORE_DIR) unless Dir.exist?(PROMETHEUS_STORE_DIR)
```

Comment:
This causes a failure:
```
Errno::ENOENT: No such file or directory @ dir_s_mkdir - /home/jenkins/workspace/test_develop_pr_core@2/database/postgresql/ruby/2.6/slave/fast/tmp/prometheus
```

```suggestion
FileUtils.mkdir_p(PROMETHEUS_STORE_DIR)
```

Or do it manually like in the secret token handling:
https://github.com/theforeman/foreman/blob/739ed809538dcc3919d6614aba44fc90b59a7ef5/config/initializers/secret_token.rb#L5-L6